.ci/atime/tests.R

```diff
@@ -19,6 +19,31 @@ for (extra.arg in extra.args.6107){
   extra.test.list[[sprintf("fread(%s) improved in #6107", extra.arg)]] <- this.test
 }
 
+# Test case adapted from https://github.com/Rdatatable/data.table/pull/4386#issue-602528139 which is where the performance was improved.
+for(retGrp_chr in c("T","F"))extra.test.list[[sprintf(
+  "forderv(retGrp=%s) improved in #4386", retGrp_chr
+)]] <- list(
+  setup = quote({
+    dt <- data.table(group = rep(1:2, l=N))
+  }),
+  expr = substitute({
+    old.opt <- options(datatable.forder.auto.index = TRUE) # required for test, un-documented, comments in forder.c say it is for debugging only.
```

Comment:
I think Jan mentioned that it is only for 'debugging' since it was not exported (coming from his [comment here](https://github.com/Rdatatable/data.table/pull/4386#issue-602528139)), but we can probably ask him for more information if we want to document it or have more details (otherwise yes, not having this option set to `TRUE` will not show performance improvement)

Reply:
Btw, good move to save the old options here and then restore them after the call to `forderv`!

Reply:
Good move to save the old options here and then restore them after the call to `forderv`!